### PR TITLE
:seedling: ignore maligned lint issue for WebhookSuite struct

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
 ignore maligned lint issue for WebhookSuite struct

### CODE REVIEW

File: pkg/plugins/golang/v3/scaffolds/internal/templates/api/webhook_suitetest.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ var _ file.Template = &WebhookSuite{}
 var _ file.Inserter = &WebhookSuite{}
 
 // WebhookSuite scaffolds the file that sets up the webhook tests
-type WebhookSuite struct {
+type WebhookSuite struct { // nolint:maligned
```
Comment (estroz) [inline pkg/plugins/golang/v3/scaffolds/internal/templates/api/webhook_suitetest.go:14]:
```suggestion
type WebhookSuite struct { //nolint:maligned
```

Comment (camilamacedo86) [reply pkg/plugins/golang/v3/scaffolds/internal/templates/api/webhook_suitetest.go]: spaced removed.

**Review by Adirio — APPROVED** [review]

**Pull request merged**